app/models/local_census_record.rb

```diff
@@ -0,0 +1,3 @@
+class LocalCensusRecord < ActiveRecord::Base
+
```

Comment:
The same question came to my mind before removing those validations from the model.

I decided to finally do it because I assumed the table would be filled without passing through Rails, and as there shouldn't be any way to modify the records once they're in the database - too sensitive data to make it easy to change it - it isn't necessary to cover those validations because they will never be used.

Anyway, if you think it makes more sense to validate the attributes also in the model to have everything covered I can add them again :)